Unable to build bundle

While attempting to release 0.11.0 #371, I receive the following error when attempting to build the bundle (`npm run bundle`):



```
created umd/mapbox-sdk.js in 620ms
Parse error at umd/mapbox-sdk.js:1478,6
      const list = errorMessages.map(function(message) {
      ^
ERROR: Unexpected token: keyword (const)
    at JS_Parse_Error.get (eval at <anonymous> (/Users/katydecorah/Documents/GitHub/mapbox/mapbox-sdk-js/node_modules/uglify-js/tools/node.js:20:1), <anonymous>:71:23)
    at fatal (/Users/katydecorah/Documents/GitHub/mapbox/mapbox-sdk-js/node_modules/uglify-js/bin/uglifyjs:291:53)
    at run (/Users/katydecorah/Documents/GitHub/mapbox/mapbox-sdk-js/node_modules/uglify-js/bin/uglifyjs:235:9)
    at Object.<anonymous> (/Users/katydecorah/Documents/GitHub/mapbox/mapbox-sdk-js/node_modules/uglify-js/bin/uglifyjs:160:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @mapbox/mapbox-sdk@0.11.0 bundle: `rollup --config ./rollup.config.js && uglifyjs umd/mapbox-sdk.js > umd/mapbox-sdk.min.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @mapbox/mapbox-sdk@0.11.0 bundle script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/katydecorah/.npm/_logs/2020-06-26T12_31_11_267Z-debug.log
```

I believe this `const` statement was introduced when @mapbox/fusspot was updated to the latest version.

@andrewharvey A couple options I'm thinking about so far, curious what you think:

1. See if we can roll back @mapbox/fusspot
2. Add babel to the the rollup.config.js to properly transform es6 features.

--- a/package.json
+++ b/package.json
@@ -78,7 +78,7 @@
     "singleQuote": true
   },
   "dependencies": {
-    "@mapbox/fusspot": "^0.7.0",
+    "@mapbox/fusspot": "^0.4.0",
     "@mapbox/parse-mapbox-token": "^0.2.0",
     "@mapbox/polyline": "^1.0.0",
     "eventemitter3": "^3.1.0",

